src/helpers/deployments/generate-deployment.js

```diff
@@ -44,4 +46,8 @@ export function generateDeployment({
       deploymentEnvironment
     }
   }
+
+  Joi.assert(deployment, deploymentValidation)
```

Comment:
is the encompassing code handling this throwing?

Reply:
No, it just throws the error and it will get handled in the standard error handling. Assumed that would be ok, but happy to discuss